x-pack/test/api_integration/apis/ml/trained_models/trained_models_list.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { USER } from '../../../../functional/services/ml/security_common';
+import { getCommonRequestHeader } from '../../../../functional/services/ml/common_api';
+
+export default ({ getService }: FtrProviderContext) => {
+  const supertest = getService('supertestWithoutAuth');
+  const ml = getService('ml');
+  const esDeleteAllIndices = getService('esDeleteAllIndices');
+
+  describe('GET trained_models_list', () => {
+    let testModelIds: string[] = [];
+
+    before(async () => {
+      await ml.api.initSavedObjects();
+      await ml.testResources.setKibanaTimeZoneToUTC();
+      testModelIds = await ml.api.createTestTrainedModels('regression', 5, true);
+      await ml.api.createModelAlias('dfa_regression_model_n_0', 'dfa_regression_model_alias');
+      await ml.api.createIngestPipeline('dfa_regression_model_alias');
+
+      // Creating an indices that are tied to modelId: dfa_regression_model_n_1
```

Comment:
Nit - `Creating an index that is tied...`

Reply:
fixed in https://github.com/elastic/kibana/pull/200977/commits/1c41a8415aa3851db2fe842c90d344e7a4c2484a